build.xml

```diff
@@ -197,13 +192,15 @@ http://forum.java.sun.com/thread.jspa?threadID=174214
 			<include name="uk/ac/open/embeddedrecorder/**"/>
     	<exclude name="uk/ac/open/embeddedrecorder/EmbeddedRecorderApp.java"/>
 		<manifest>
-			<attribute name="Permissions" value="{jarpermissions}"/>
-			<attribute name="Caller-Allowable-Codebase" value="{allowablecode}"/>
+			<attribute name="Permissions" value="all-permissions"/>
+ 			<attribute name="Caller-Allowable-Codebase" value="*.open.ac.uk *.open.edu *.vledev3.open.ac.uk/"/>
```

Comment:
Hardcoding this one means nobody outside the OU can use this build script - I don't suppose anyone actually does though so maybe it doesn't matter...

Reply:
Have not changed